Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -397,6 +400,9 @@ public struct TargetSourcesBuilder {
                 }
             }
 
+            // Ignore README files.
+            if path.basename.hasPrefix("README") { continue}
```

Comment:
Should we accommodate differences in case (e.g. `readme.md`)?

```suggestion
            if path.basenameWithoutExt.uppercased() == "README" { continue }
```

Reply:
Maybe. I'm a bit uncertain that this is the right thing to do in any way whatsoever. Maybe it would be better to detect cases where there are C(++)/ObjC(++) sources and ignore any of the file formats we support for literate Swift.